dspace/config/modules/iiif.cfg

```diff
@@ -0,0 +1,93 @@
+
+# Optional image to associate with manifests. A small image that represents
+# an individual or organization associated with the resource it is attached to.
+# iiif.logo.image = https://image/url/i.png
+
+# Only these origins (client URLs) can successfully communicate with the IIIF API. This
+# allows XHR requests from remote IIIF clients. Defaults to ${dspace.ui.url} if unspecified
+# (as the embedded IIIF client must have access to the API). Multiple allowed origin URLs may
+# be comma separated. Wildcard value (*) is NOT SUPPORTED. # (Requires reboot of servlet
+# container, e.g. Tomcat, to reload)
+iiif.cors.allowed-origins = ${dspace.ui.url}
+
+# Whether or not to allow credentials (e.g. cookies) sent by the client/browser in CORS
+# requests (in "Access-Control-Allow-Credentials" header).
+# For the DSpace iiif endpoint, we default this to "false" .
+# (Requires reboot of servlet container, e.g. Tomcat, to reload)
+iiif.cors.allow-credentials = false
+
+# metadata to include at the resource level in the manifest
+# labels are set in the Messages.properties i18n file
+iiif.metadata.item = dc.title
+iiif.metadata.item = dc.date.issued
+iiif.metadata.item = dc.contributor.*
+iiif.metadata.item = dc.description.abstract
+
+# metadata to be added to the canvas from the bitstream, labels are set in 
+# the Messages.properties i18n file. It is possible to include the technical
+# information stored in the bitstream format registry and in the checksum using the placeholder
+# @format@ to include the short description of the bitstream format as entered by the user or 
+# stored in the registry
+# @mimemtype@ to include the mimetype associated with the bitstream format in the registry
+# @checksum@ to include the computed checksum for the file and the used algorithm
+# @bytes@ to include the size of the images in bytes
+iiif.metadata.bitstream = dc.title
+iiif.metadata.bitstream = dc.description
+iiif.metadata.bitstream = @format@
+iiif.metadata.bitstream = @mimetype@
+iiif.metadata.bitstream = iiif.image.width
+iiif.metadata.bitstream = iiif.image.height
+iiif.metadata.bitstream = @bytes@
+iiif.metadata.bitstream = @checksum@
+
+# the metadata to use to provide machine readable information about the resource right usage
+iiif.license.uri = dc.rights.uri
+
+# static text to be used as attribution in the iiif manifests
+iiif.attribution = ${dspace.name}
+
+iiif.logo.image = ${dspace.ui.url}/assets/images/dspace-logo.svg
+# (optional) one of individuals, paged or continuous. Can be overridden at the item level via 
+# the iiif.view.hint metadata 
+iiif.document.viewing.hint = individuals
+
+
+
+# public base url of a iiif server able to serve DSpace images. The bitstream uuid is appended to this URL 
+iiif.image.server = 
+ 
+# these should be not changed. They must match the routing configuration for the iiif controller 
+iiif.url = ${dspace.server.url}/iiif
+iiif.solr.search.url =
+iiif.bitstream.url = ${dspace.server.url}/api/core/bitstreams
```

Comment:
Silly question: If these configurations should not be changed, then why are they configurations?  Shouldn't they be constants in the code somewhere?  Also unclear why `iiif.solr.search.url` is set to empty.

Reply:
I'll look into this. It seems that they should be in code as you suggest.